opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -260,15 +260,22 @@ def test_invalid_instrumentation_info(self):
         self.assertEqual(
             tracer1.instrumentation_info, tracer2.instrumentation_info
         )
+
         self.assertIsInstance(
             tracer1.instrumentation_info, InstrumentationInfo
         )
         span1 = tracer1.start_span("foo")
         self.assertTrue(span1.is_recording())
         self.assertEqual(tracer1.instrumentation_info.version, "")
-        self.assertEqual(
-            tracer1.instrumentation_info.name, "ERROR:MISSING MODULE NAME"
+        self.assertEqual(tracer1.instrumentation_info.name, "")
+
+        self.assertIsInstance(
+            tracer2.instrumentation_info, InstrumentationInfo
         )
+        span2 = tracer2.start_span("bar")
+        self.assertTrue(span2.is_recording())
+        self.assertEqual(tracer2.instrumentation_info.version, "")
+        self.assertEqual(tracer2.instrumentation_info.name, "")
```

Comment:
Are these repeating the test in line 260-261?

Reply:
Good point. Perhaps the assertEqual test in lines 260-262 should be moved after testing both tracers individually (after line 279)? @xukaren

Reply:
I meant that it seems it's doing the same thing as L260-261, it seems this could be removed, but up to you :)